CHANGELOG.md

```
@@ -1,15 +1,25 @@
-#### 9.4.18 - October 2024
+#### 9.5.2 - October 2024
   * Added new `FieldEnum` with all the available field names
   * Code security improvements as per CodeQL recommendations
 
-#### 9.4.17 - June 2024
+#### 9.5.1 - June 2024
   * (PW-1913) Added IBAN validation for Egypt local account structure
+  * Restore deprecated method in MT210 class
+
+#### 9.5.0 - May 2024
+  * SWIFT Standard release update 2024 (live 16 November 2025)
```

Markdownlint

Expected: 0; Actual: 2
Unordered list indentation

(MD007, ul-indent)